### DISCUSSION TITLE
Can not access to all function owners (discounts) which are create with same functionId

### DISCUSSION THREAD

**Discussion by DunggNguyenn** [Shopify/function-examples#211] [Feedback: Discounts APIs]
I need to create multiple discounts with the same functionId but they have different metafield value. 
So I need to access the metafield value of all these discount nodes via **input.graphql**

If I create only one discount, the metafield value has a limit is 1KB. So I can not use this solution. 

Please give me some advice. Thank you!

**Comment by DuncanUszkay1** [comment]
>If I create only one discount, the metafield value has a limit is 1KB.

You don't need to create multiple discounts, you can create multiple metafields on the same discount and load them in your function. 

E.G. in your `input.graphql` file:
```
functionConfiguration1: metafield(namespace: "priority-locations", key: "config_1") { value },
functionConfiguration2: metafield(namespace: "priority-locations", key: "config_2") { value }
...
```

Hope this helps!
